stdcommands/roast/roastlist.go

```diff
@@ -161,4 +161,119 @@ var roasts = []string{
 	"You're the reason the gene pool needs a lifeguard.",
 	"I don't know what your problem is, but I'm guessing it's hard to pronounce.",
 	"Talking to you is like stepping on a leaf in autumn and hearing no crunch- disappointment.",
+        "You do realize we're just tolerating you, right?",
+        "Your birth certificate should be rewritten as a letter of apology.",
+        "If I were a dog and you were a flower, I'd lift my leg up and give you a shower.",
+        "You just might be why the middle finger was invented in the first place.",
+        "Your so annoying you could make a happy meal cry.",
+        "You’re the reason they invented the dewey decimal system they needed to categorise your ass on mass.",
+        "If you ever feel like you’ve made a bad mistake remember your parents made you.",
+        "You're as useless as Sophie Turner's stunt double in Game of Thrones.",
+        "You're like a flat soda, once a promising fizz, now just a disappointment.",
+        "You're so slow, even a sloth would tell you to pick up the pace.",
+        "You're so simple, you'd be surpirsed to learn that strippers take their clothes off.",
+        "I bet you think virginity is just a star sign.",
+        "If you asked your reflection out and it will say \"oh fuck no you ugly as heeeeelllllllll\".",
+        "Somehow, you're more useless than a chocolate teapot.",
+        "Are you a magician? Everytime you step into a room everyone disappears.",
+        "You're as handy as a pocket on the back of a shirt.",
+        "You're as usefull as a parachue during a bungie jump.",
+        "You're as much of a bright spark as a black hole.",
+        "Even a tortoise on tranquillisers can think faster then you.",
+        "You're as wanted as discord community guideline updates.",
+        "You're about as interesting as an end user licence agreement.",
+        "I feel sorry for your chair.",
+        "Expired medicine is more beneficial then your existence.",
+        "A cracked mug is less fragile then your ego",
+        "You're as scary as a kitten in a pet shop, and as needy as one too.",
+        "Your about as useful as a square wheel.",
+        "We don't have to ask if your mother drank during your pregnacy, that much is clear.",
+        "You explain every joke you make don't you?",
+        "Your birth certificate is an apology letter and compensation from the condom factory.",
+        "You look like Steven Hawking talked.",
+        "You look how wet concrete feels to walk on.",
+        "Bees choose not to sting you, you're not worth it.",
+        "You're about as menacing as a jelly-bean sized baby kangaroo.",
+        "If anyone requires an off button, it's you.",
+        "You haven't changed since the last time I saw you. You really should.",
```

Comment:
You're

Reply:
I'm not seeing this one

Reply:
my bad, I was reviewing on phone and `you really should` as `you really stupid`